### PR TITLE
Rhel8

### PR DESCRIPTION
**Pull request by ndokos**

### CODE REVIEW

File: configtools.spec [hunk 1]
```diff
@@ -4,11 +4,16 @@ Release:     1
...
 
 License:     GPLv3+
-URL:         http://pypi.python.org/pypi/%{name}
-Source0:     http://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
+URL:         https://pypi.python.org/pypi/%{name}
+Source0:     https://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
 Buildarch:   noarch
 
+%if 0%{?rhel} == 7
 BuildRequires:	python2-devel
```
Comment (portante) [inline configtools.spec]: Aren't we all in on Python3 now?

**Review by portante — APPROVED** [review]

File: configtools.spec (hunk 1 above)
Comment (ndokos) [reply configtools.spec]: Yes - I wasn't sure that RHEL7 could do it, but I did some testing and it seems to work properly (as long as we can get python3 and python3-devel RPMs which are in the RHEL7-latest repos).

**Comment by ndokos** [comment]
@portante: can you re-review this? Got rid of python2 traces and rebases after the PR #16 merge. Thanks!

**Review by portante — APPROVED** [review]
Looks good!

**Pull request merged**